src/components/main-page/Schedule/config.ts

```diff
@@ -0,0 +1,48 @@
+type ScheduleTimer = {
+  id: number;
+  startTimestamp: Date;
+  endTimestamp: Date;
+};
+
+export const scheduleList: ScheduleTimer[] = [
+  {
+    id: 1,
+    startTimestamp: new Date(2022, 9, 7, 23, 30),
```

Comment:
I need a suggestion - should I use here `new Date()` or `dayjs`? And how about timezones here? If it is rendered on server, it takes server time? Should I consult fixed GMT+2 offset?

Reply:
In `new Date()` approach there are some tricky things, because you have e.g. months indexed from 0. But if you wrote something wrong, Typescript would detect that. In `dayjs` as far as I know you give only a string - it's more readable, but any tool would not detect for example wrong date format.